swarm/network/stream/snapshot_sync_test.go

```diff
@@ -144,29 +90,186 @@ func TestSyncing(t *testing.T) {
 	}
 }
 
-//Do run the tests
-//Every test runs 3 times, a live, a history, and a live AND history
+func TestSyncingViaRPC(t *testing.T) {
```

Comment:
should not be called RPC cos it should not use RPC either

Reply:
Why not? The idea of this test is to compare syncing by calling directly the (manual) subscribe method implemented in the test, how to call it if not via RPC?